docs/index.rst

```diff
@@ -137,6 +215,17 @@ to attach media to statuses.
 
 .. automethod:: Mastodon.media_post
 
+Returns a media dict:
+.. code-block:: python
+    mastodon.media_post("image.jpg", "image/jpeg")
+    # Returns
+    {
+     'text_url': The display text for the media (what shows up in toots)
+     'preview_url': The URL for the media preview on Amazon S3
```

Comment:
Admins can choose whether to use S3 or not. Typically it's just local file storage. In either case, this is a URL to the preview image file.